packages/xrpl/src/models/transactions/XChainAccountCreateCommit.ts

```diff
@@ -0,0 +1,104 @@
+import { ValidationError } from '../../errors'
+import { Amount, XChainBridge } from '../common'
+
+import {
+  BaseTransaction,
+  isAmount,
+  isXChainBridge,
+  validateBaseTransaction,
+} from './common'
+
+/**
+ * The XChainAccountCreateCommit transaction creates a new account on one of the
+ * chains a bridge connects, which serves as the bridge entrance for that chain.
```

Comment:
I'm not quite sure what this means by "which serves as the bridge entrance for that chain".
```suggestion
 * chains a bridge connects, which serves as the bridge entrance for that chain.
```

Reply:
The suggested change doesn't look it has any diff.
I'm copying from the docs here: https://opensource.ripple.com/docs/xls-38d-cross-chain-bridge/transaction-types/xchainaccountcreatecommit/

Reply:
Sorry, I was more trying to highlight the part that wwas confusing for me specifically - I don't know what the better wording would be since I'm not sure what it's trying to say.

Reply:
cc @oeggert